src/tabwidget.cpp

```diff
@@ -244,7 +244,7 @@ void TabWidget::contextMenuEvent(QContextMenuEvent *event)
     } else if (action == rename) {
         emit tabRenameRequested(tabIndex);
     } else if (action == changeColor) {
-	emit tabTitleColorChangeRequested(tabIndex);
+    emit tabTitleColorChangeRequested(tabIndex);
```

Comment:
Indentation messed up, too.

Reply:
I'll add a commit to reindent the whole file.